xml/net_wicked.xml

```diff
@@ -700,46 +642,83 @@ ip link set down dev eth0
     <command>list</command>).
    </para>
    <para>
-    Change the state of a device with the command <command>ip link
-    set</command>&nbsp;<option><replaceable>DEVICE_NAME</replaceable></option>&nbsp;<command/>
-    . For example, to deactivate device eth0, enter <command>ip link
-    set</command> <option>eth0 down</option>. To activate it again, use
-    <command>ip link set</command> <option>eth0 up</option>.
+    Change the state of a device with the command:
    </para>
+<screen>&prompt.sudo;ip link set <replaceable>DEV_NAME</replaceable></screen>
+   <para>
+    For example, to deactivate device eth0, enter
+    </para>
+<screen>&prompt.sudo;ip link set eth0 down</screen>
+   <para>
+    To activate it again, use
+   </para>
+   <tip>
+    <title>Disconnecting NIC Device</title>
+    <para>
+     If you deactivate a device with
+     </para>
+<screen>&prompt.sudo;ip link set <replaceable>DEV_NAME</replaceable> down</screen>
+    <para>
+     it disables a number of its internal features, but the device remains
+     connected to the physical network environment. To disconnect the device
+     completely, run
+    </para>
+<screen>&prompt.sudo;ip link set <replaceable>DEV_NAME</replaceable> carrier off</screen>
```

Comment:
This sounds a bit as if `ip link set DEV carrier off` was stronger (or did more) than `ip link set DEV down` but they are in fact independent. While `down` disables the network interface on software level, `carrier off` simulates losing the link (like when ethernet cable is unplugged or connected switch is turned off). So e.g. `ip link set DEV down` would drop all routes using `DEV` but `ip link set DEV carrier off` does not.

It should be also mentioned that `carrier off` requires support from the network device driver (and only some drivers provide the support).